boards/arm/rp2040/raspberrypi-pico/src/rp2040_bringup.c

```diff
@@ -102,5 +102,17 @@ int rp2040_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_BMP180
+  /* Try to register BMP180 device in I2C0 */
+
+  ret = board_bmp180_initialize(0);
+
```

Comment:
This blank line can also be removed

Reply:
@gustavonihei , thanks for the review. I've just fixed it. Please, would you review it again?